context-based-restrictions/v1.ts

```diff
@@ -39,7 +39,7 @@ import { getSdkHeaders } from '../lib/common';
  */
 
 class ContextBasedRestrictionsV1 extends BaseService {
-  static DEFAULT_SERVICE_URL: string = 'https://cbr.cloud.ibm.com';
+  static DEFAULT_SERVICE_URL: string = 'https://cbr.test.cloud.ibm.com';
```

Comment:
You must have used the staging version of the API definition when you generated this code.  I didn't see this change in the other PRs.   Please re-gen with the same version of the API definition that you used for the other languages.